definitions/checks/disk/available_space.rb

```diff
@@ -0,0 +1,23 @@
+module Checks
+  module Disk
+    class AvailableSpace < ForemanMaintain::Check
+      metadata do
+        label :available_space
+        description 'Check if root(/) partition has enough space to proceed with upgrade'
+        tags :default
+      end
+
+      MIN_SPACE_IN_MB = 4096
+
+      def run
+        assert(!enough_space?, "System has less than #{MIN_SPACE_IN_MB / 1024}GB space available"\
+              ' on root(/) partition')
+      end
+
+      def enough_space?
```

Comment:
By name of the method 'enough_space?', anyone expects that this method should return true when there is enough same.
Currently, it is returning nil for positive scenario. 
Please make a correction to method and call `enough_space?` in assert.